includes/delete_datastream.form.inc

```diff
@@ -20,6 +20,11 @@
  */
 function islandora_delete_datastream_form(array $form, array &$form_state, AbstractDatastream $datastream) {
   $form_state['datastream'] = $datastream;
+  $form['delete_derivatives'] = array(
+    '#title' => t('Delete Derviatives'),
+    '#type' => 'checkbox',
+    '#default_value' => 0,
+  );
```

Comment:
Could/should probably list the datastreams which would be purged, so people know what will happen?